content/en/docs/reference/version-policy.md

```diff
@@ -112,56 +112,55 @@ documentation for that application.
           (<a href="https://github.com/kubeflow/katib">GitHub</a>)
           </td>
         <td>Beta</td>
-        <td>v1alpha3</td>
+        <td>v0.12.0</td>
       </tr>
       <tr>
         <td><a href="/docs/components/kfserving/kfserving/">KFServing</a>
           (<a href="https://github.com/kubeflow/kfserving">GitHub</a>)
         </td>
         <td>Beta</td>
-        <td>v0.5.1</td>
+        <td>v0.6.0</td>
       </tr>
       <tr>
         <td><a href="/docs/components/training/mpi/">MPI training: MPI 
           operator</a>
           (<a href="https://github.com/kubeflow/mpi-operator">GitHub</a>)
         </td>
         <td>Alpha</td>
-        <td></td>
+        <td>v0.3.0</td>
```

Comment:
I updated API versions in this PR: https://github.com/kubeflow/website/pull/3002, some of the components became stable.
@shannonbradshaw Is it fine for you ?